Entries/Copy-X/Copy-X.md

```diff
@@ -0,0 +1,283 @@
+
+* HeroponRikiBestest - Entry author
+
+***
+
+## Table of Contents:
+
+* [Entry Information](#entry-information) ([Alt.](#Entry%20Information))
+	* [DRML Entry Name](#drml-entry-name) ([Alt.](#DRML%20Entry%20Name))
+	* [Authors/Contributors](#authors/contributors)
+	* [Table of Contents](#table-of-contents) ([Alt.](#Table%20of%20Contents))
+	* [Categories](#categories)
+	* [DRML Entry Version](#drml-entry-version) ([Alt.](#DRML%20Entry%20Version))
+* [DRM Information](#drm-information) ([Alt.](#DRM%20Information))
+	* [Names](#names)
+	* [Developers](#developers)
+	* [Protection Features](#protection-features) ([Alt.](#Protection%20Features))
+	* [Overall Description](#overall-description) ([Alt.](#Overall%20Description))
+	* [Timeline](#timeline)
+	* [Versions & Differences](#versions--differences) ([Alt.](#Versions%20&%20Differences))
+		* [Tested](#tested)
+		* [Untested](#untested)
+	* [Samples](#samples)
+		* [Confirmed](#confirmed)
+		* [Unconfirmed](#unconfirmed)
+	* [Supported Platforms](#supported-platforms) ([Alt.](#Supported%20Platforms))
+	* [Software Known to Detect](#software-known-to-detect) ([Alt.](#Software%20Known%20to%20Detect))
+	* [Preservation Instructions](#preservation-instructions) ([Alt.](#Preservation%20Instructions))
+	* [Associated File Attributes](#associated-file-attributes) ([Alt.](#Associated%20File%20Attributes))
+	* [Known Manufacturing Information](#known-manufacturing-information) ([Alt.](#Known%20Manufacturing%20Information))
+		* [Known Manufacturing Plants](#known-manufacturing-plants) ([Alt.](#Known%20Manufacturing%20Plants))
+		* [Known Ringcode Information](#known-ringcodeinformation) ([Alt.](#Known%20Ringcode%20Information))
+			* [Mastering Code](#mastering-code)  ([Alt.](#Mastering%20Code))
+			* [Mastering SID Code](#mastering-sid-code)  ([Alt.](#Mastering%20SID%20Code))
+			* [Toolstamp or Mastering Code](#toolstamp-or-mastering-code)  ([Alt.](#Toolstamp%20or%20Mastering%20Code))
+			* [Mould SID Code](#mould-sid-code)  ([Alt.](#Mould%20SID%20Code))
+* [Reference Material](#reference-material) ([Alt.](#Reference%20Material))
+	* [Media](#media)
+	* [Additional Resources](#additional-resources) ([Alt.](#Additional%20Resources))
+	* [Footnotes](#footnotes)
+
+*** 
+
+## Categories: 
+
+* Optical Copy Protection
+
+***
+
+## Protection Features:
+
+* Ring protection
+* Dummy files
+* Disc check (Professional)
+* SoftICE Debugger Protection (Professional)
+
+***
+
+## DRML Entry Version: 
+
+***
+
+# DRM Information:
+
+## Names: 
+
+* copy-X[^CopyX_Original_Website]
+* copy-X - the ultimate Copy-Killer.[^CopyX_Original_Website]
+* copy-X audio[^CopyX_Original_Website]
+* copy-X light[^CopyX_Original_Website]
+* copy-X professional[^CopyX_Original_Website]
+* copy-X trial maker[^CopyX_Original_Website]
+
+[^CopyX_Original_Website]: [The original optimal media link for copy-X.](https://web.archive.org/web/20011016234742/http://www.optimal-online.de:80/product/copy_x.htm)
+
+***
+
+## Developers:
```

Comment:
the "Current Website" footnote is currently next to the original name. Please move the footnote to the current name, and add the footnote for the original website to original name. If possible, add a source for the name changing, or at least an archive of the website with the new name from 2011.

Reply:
Fixed.